arm-datalake-analytics/catalog/2016-11-01/swagger/catalog.json

```diff
@@ -828,6 +828,92 @@
         "x-ms-odata": "#/definitions/USqlTable"
       }
     },
+    "/catalog/usql/databases/{databaseName}/schemas/{schemaName}/statistics": {
+      "get": {
+        "tags": [
+          "Catalog"
+        ],
+        "operationId": "Catalog_ListStatisticsByDatabaseAndSchema",
```

Comment:
This is a good question. I am hesitant to make a unique group for each catalog item type (there are 16 types and we are adding more all the time) that will have two or three methods: Get, List and optionally ListBy. This would result in a massive set of client operation classes:
DatabaseOperations
SchemaOperations
TableOperations
TableTypeOperations
TypeOperations
ViewOperations
... etc.

If you think this is the better way to go, though, in the long run, I am amenable to changing it, but it is a pretty large breaking change for our existing tools (sdks, powershell cli, etc).